src/main/org/firebirdsql/gds/ng/FbStatement.java

```diff
@@ -227,6 +227,13 @@
      */
     String getExecutionPlan() throws SQLException;
 
+    /**
+     * @return The detailed execution plan of the currently prepared statement
+     * @throws SQLException
+     *         If this statement is closed.
+     */
+    String getExplainedExecutionPlan() throws SQLException;
```

Comment:
Although it matches the name in the Firebird 3 release notes, I'm not sure I like the name 'explained plan'. I might change it after accepting the merge.

It's also possible that I'll change this to a method `getExecutionPlan(PlanType)` where `PlanType` would be an enum that has both types.

NOTE: Just observation, no need to change this.